src/main/deploy/constants/BlueFieldLocations.json

```diff
@@ -62,6 +62,13 @@
   "blueCoralStationLeftRotation": {
     "radians": -1.0471975511965976
   },
+  "blueNetTranslation": {
+    "x": 10.319,
+    "y": 2.768
```

Comment:
noting this is the incorrect field location just so we don't randomly get cooked by this at comp